neurodamus/node.py

```diff
@@ -1865,50 +1866,54 @@ def _coreneuron_restore(self):
         self._sim_corenrn_write_config(corenrn_restore=True)
         self._sim_ready = True
 
-    # -
-    def _instantiate_simulation(self):
-        # Keep the initial RSS for the SHM file transfer calculations
-        self._initial_rss = SHMUtil.get_node_rss()
-        print_mem_usage()
-
-        self.load_targets()
-
-        # Check connection block configuration and raise warnings for overriding
-        # parameters
-        SimConfig.check_connections_configure(self._target_manager)
+    @property
```

Comment:
Why use @property? Here it does not simply get the value of an attribute, and n_cycles is not an attribute of the class either.

You could call `n_cycles()` later on.

Reply:
Tristan always wanted properties for functions that return a value and have no inputs. I have no strong opinion

Reply:
done